conditional/src/main/java/com/linecorp/conditional/ConditionMatchResult.java

```diff
@@ -184,8 +202,10 @@ public String toString() {
                .append(", thread=").append(thread.getName())
                .append(", delay=").append(millisAsString(condition.delayMillis()))
                .append(", timeout=").append(millisAsString(condition.timeoutMillis()))
-               .append(", startTime=").append(millisAsString(startTimeMillis))
-               .append(", endTime=").append(millisAsString(endTimeMillis))
+               .append(", startTime=").append(startTime())
+               .append(", endTime=").append(endTime())
+               .append(", startTimeMillis=").append(millisAsString(startTimeMillis))
+               .append(", endTimeMillis=").append(millisAsString(endTimeMillis))
```

Comment:
If startTime and endTime are displayed in ISO-8601 format, it would be good to remove startTimeMillis and endTimeMillis.

Reply:
I applied the way you suggested at a7c1734 🙂